### PR TITLE
Merge template version 3.0.2

### PR DESCRIPTION
**Pull request by itrujnara**
Yet another template merge.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] Make sure your code lints (`nf-core lint`).

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ If you're not used to this workflow with git, you can start with some [docs from
 You have the option to test your changes locally by running the pipeline. For receiving warnings about process selectors and other `debug` information, it is recommended to use the debug profile. Execute all the tests with the following command:
 
 ```bash
-nextflow run . -profile debug,test,docker --outdir <OUTDIR>
+nf-test test --profile debug,test,docker --verbose
```
Comment (mashehu) [inline .github/CONTRIBUTING.md]:
```suggestion
nextflow run . -profile debug,test,docker --outdir <OUTDIR>
```
I don't think this pipeline uses nf-test yet

**Review by mashehu — APPROVED** [review]

File: CITATIONS.md [hunk 2]
```diff
@@ -56,6 +56,10 @@
 
 > Vincent Lefort, Richard Desper, Olivier Gascuel, FastME 2.0: A Comprehensive, Accurate, and Fast Distance-Based Phylogeny Inference Program, Molecular Biology and Evolution, Volume 32, Issue 10, October 2015, Pages 2798–2800, https://doi.org/10.1093/molbev/msv150
 
+- [MultiQC](https://pubmed.ncbi.nlm.nih.gov/27312411/)
+
+> Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016 Oct 1;32(19):3047-8. doi: 10.1093/bioinformatics/btw354. Epub 2016 Jun 16. PubMed PMID: 27312411; PubMed Central PMCID: PMC5039924.
```
Comment (maxulysse) [inline CITATIONS.md]:
```suggestion
  > Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016 Oct 1;32(19):3047-8. doi: 10.1093/bioinformatics/btw354. Epub 2016 Jun 16. PubMed PMID: 27312411; PubMed Central PMCID: PMC5039924.
```

**Review by maxulysse — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (itrujnara) [reply .github/CONTRIBUTING.md]: Not yet, it might when we have the time

**Review by maxulysse — APPROVED** [review]

**Comment by itrujnara** [comment]
Uniprot is currently having issues, will check again in a few hours

**Comment by itrujnara** [comment]
The pull_request_target check seems broken, merging despite it

**Pull request merged**